### ISSUE TITLE
Error when editing tags of some songs

### ISSUE THREAD

**Issue by dspacejs** [Taiko2k/TauonMusicBox#572]
When I try and edit the metadata of some songs, I get the following error:

```
Cannot handle multi-field! Please use external tag editor
```

I'm not sure why this only happens with some files and not others. And the fact it works with other tag editors suggests this is probably a bug with Tauon.

**Comment by Taiko2k** [comment]
Yeah its a current limitation that it can't handle tracks that have multiple artists/genres. But it something that should be fixed, eventually.
